src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Solution/CurrentProjectSelectionStep.cs

```diff
@@ -80,7 +80,7 @@ protected override async Task<UpgradeStepInitializeResult> InitializeImplAsync(I
             var completeChecks = _orderedProjects.Select(async p => IsCompleted(context, p) || !await RunChecksAsync(p, token).ConfigureAwait(false));
```

Comment:
@mjrousos While you are here, could you address the PR build failures we are seeing - https://dev.azure.com/dnceng/public/_build/results?buildId=1041902&view=logs&j=396e4757-3dc0-597a-e469-5feb3123c7be&t=44c646e6-db1b-5521-5b76-c5c2c57c3a93&l=83 - add a suppression for context.entrypoint check in LN 65.

Reply:
Yep, that's what I'm doing in this file. Unless I'm mistaken, the build break came from line 83 checking `entryPoint` for null unnecessarilly, which this change addresses.

Reply:
Ah perfect, I missed the removal of ? :)

Reply:
Yay! I'm glad there's a check for that.

Reply:
Weird that it just started happening w/o a change that got blocked in CI, though

Reply:
Yea, I thought the same thing. This broke when I added the check for EntryPoint not being null in #351. It's weird it didn't fail then. Who knows.